src/main/java/Domain/Exceptions/ItemOutOfStockExepction.java

```diff
@@ -0,0 +1,12 @@
+package Domain.Exceptions;
+
+public class ItemOutOfStockExepction extends RuntimeException {
```

Comment:
// This class represents an error when a product is out of stock.

change to product- all the classes are product and not item pls

Reply:
ok